ChangeLog.md

```diff
@@ -18,6 +18,9 @@ Bug fixes:
 * Run the Cabal file checking in the `sdist` command more reliably by
   allowing the Cabal library to flatten the
   `GenericPackageDescription` itself.
+* Benchmarks used to be run concurrently with other benchmarks
```

Comment:
I'm assuming the reason for this is to not confuse the benchmark results with other CPU activities, right? May be worth pointing that out.

Reply:
Yep, done!